### PR TITLE
New cred param to control s3 path style access.

### PR DESCRIPTION
**Pull request by talakad**
Signed-off-by: veda <veda@portworx.com>

**What this PR does / why we need it**:
Allows to access both virtual-host-style and path-style access to s3 and allows users to control that as S3 providers may not support both.
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -2189,6 +2189,8 @@ message SdkAwsCredentialRequest {
   string region = 4;
   // (optional) Disable SSL connection
   bool disable_ssl = 5;
+  // (optional) Disable path-styl access
```
Comment (lpabon) [inline api/api.proto]: nit: `styl` -> `style`

File: api/api.proto [hunk 2]
```diff
@@ -2217,6 +2219,8 @@ message SdkAwsCredentialResponse {
...
```
Comment (lpabon) [inline api/api.proto]: same here

File: api/api.proto [hunk 3]
```diff
@@ -3296,7 +3300,7 @@ message SdkVersion {
     // SDK version major value of this specification
     Major = 0;
     // SDK version minor value of this specification
-    Minor = 55;
+    Minor = 57;
```
Comment (lpabon) [inline api/api.proto]: nice

File: SDK_CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 ## Releases
 
+### v0.57.0 - Tech Preview (6/4/2019)
+
+* Added new param to control path style access to s3
+*
```
Comment (lpabon) [inline SDK_CHANGELOG.md]: I think there is an extra bullet here.

File: SDK_CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 ## Releases
 
+### v0.57.0 - Tech Preview (6/4/2019)
+
+* Added new param to control path style access to s3
```
Comment (lpabon) [inline SDK_CHANGELOG.md]: May want to mention that this affects the credentials Api

File: api/api.proto (hunk 1 above)
Comment (talakad) [reply api/api.proto]: done.

File: SDK_CHANGELOG.md (hunk 5 above)
Comment (talakad) [reply SDK_CHANGELOG.md]: Yup. took it out.

**Pull request merged**